1

pavlovasofia65 opened the pull request.

pavlovasofia65 committed:
1

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,26 @@
+    <div class="container-cu">
+        <div class="forst-div">
+            <h2 class="header-cu">Contact Us</h2>
+            <form action="" class="form-cu">
```
Атрибут "action" є зайвий.

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,26 @@
+        <div class="forst-div">
+            <h2 class="header-cu">Contact Us</h2>
+            <form action="" class="form-cu">
+                <p class="text-cu">Please use the form below to contact us. Enter your name, email, phone, and message and we'll get back to you shortly.</p>
```
Параграф має бути перед тегом "form" і не є частиною цього тегу.

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,26 @@
+                    <input name="user-name" type="text" class="input-cu" placeholder="Name">
+                </label>
+                <label class="label-cu">
+                    <input name="user-email" type="mail" class="input-cu" placeholder="Email" pattern="([A-z0-9_.-]{1,})@([A-z0-9_.-]{1,}).([A-z]{2,8})">
```
Має бути type="Email, a  не "type="mail"

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,26 @@
+                <label class="label-cu" >
+                    <input name="user-phone" type="tel" class="input-cu" placeholder="+380" pattern="^\+\d{3}-\d{2}-\d{3}-\d{2}-\d{2}$">
+                </label>
+                <textarea name="comment" id="" class="textarea-cu" placeholder="Comment">
```
id="" не може бути пустим

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,26 @@
+            </form>
+        </div>
+
+        <div class="second-div"><img src="/img/contact-us/contact-us-mob.png" alt="watch" class="image-cu" srcset="/img./contact-us./contact-us-mob.png 1x, /img./contact-us./contact-us-mob@2x.png 2x">
```
Зображення мають бути реалізовані за допомогою тегу picture: 

<picture>
                <source media="(min-width: 1280px)" srcset="/img/contact-us/contact-us-desk.png 1x, /img/contact-us/contact-us-desk@2x.png 2x"
                width="430"
                height="556"/>
  
                <source media="(min-width: 768px)" srcset="/img/contact-us/contact-us-tab.png 1x, /img/contact-us/contact-us-tab@2x.png 2x"
                width="704"
                height="556"/>
  
                <source media="(max-width: 767px)" srcset="/img/contact-us/contact-us-mob.png 1x, /img/contact-us/contact-us-mob@2x.png 2x"
                width="335"
                height="512"/>
  
                <img class=""
                src="/img/contact-us/contact-us-desk.png"
                width="360"
                height="300"
                alt=""
              />
              </picture>

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,26 @@
+<section class="contact-us" id="contact-us">
+    <div class="container-cu">
```
Перший div має мати клас під назвою "container", а не "container-cu", щоб застосувались загальні стилі з файлу container.css. Після нього, йде заголовок і тільки далі, наступний div з класом "first-div", який огортає параграф (тег <p>) та форму.

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,26 @@
+<section class="contact-us" id="contact-us">
+    <div class="container-cu">
+        <div class="forst-div">
```
До div з класом "first-div" також застосовані стилі (мобільна версія):
border-radius: 30px;
padding: 48px 28px;
width: 335px;
height: 512px;
background: #f3f3f3;

anakovalchuk commented on src/css/contact-us.css:
```diff
@@ -0,0 +1,207 @@
+margin-bottom: 24px;
+}
+
+.form-cu {
```
ці стилі задаються для div, не формі

anakovalchuk reviewed (changes requested):
Привіт! Коментарі додала до деяких рядків. В загальному ще потрібно допрацювати над кодом та стилями (зустрічається дублювання базових стилів, зокрема). Перед повторним відправленням перевір код та стилі через валідатор. Дякую)

pavlovasofia65 committed:
2

pavlovasofia65 committed:
3

Привіт, трошки підчистила код, додала тег pictures, p i form огорнула в div + ще декілька налаштувань. Подивись, будь ласка, що не так із зображеннями, не хочуть відображатися(
Дякую!

pavlovasofia65 committed:
4

anakovalchuk reviewed (changes requested):
<img width="218" alt="Screenshot 2024-09-08 at 15 27 34" src="https://github.com/user-attachments/assets/5c53081b-0817-41a6-8ebd-a281f60b4905">

Зверни увагу, як себе поводить форма, коли стає меншою ніж 375px. Скоріше за все діву  .form-div, потрібно задати min-width: 335px, а не max.

Також зверни увагу на висоту інпутів, вони на декілька пікселів відрізняються від того, що є на макеті, можливо для класу .input-cu треба задати height згідно даних макету)

pavlovasofia65 committed:
3

pavlovasofia65 committed:
4

pavlovasofia65 committed:
5

anakovalchuk commented on src/partials/contact-us.html:
```diff
@@ -0,0 +1,43 @@
+        </div>
+
+        <div class="second-div-cu">
+            <picture>
```
Розмітка для зображень прописана не вірно. Спробуй замінити код:


    <picture>
                <source media="(min-width: 1280px)" srcset="/img/contact-us/contact-us-desk.png 1x, /img/contact-us/contact-us-desk@2x.png 2x"
                width="530"
                height="556"/>
        
                <source media="(min-width: 768px)" srcset="/img/contact-us/contact-us-tab.png 1x, /img/contact-us/contact-us-tab@2x.png 2x"
                width="704"
                height="556"/>
        
                <source media="(max-width: 767px)" srcset="/img/contact-us/contact-us-mob.png 1x, /img/contact-us/contact-us-mob@2x.png 2x"
                width="335"
                height="512"/>
        
                <img class="image-cu"
                src="/img/contact-us/contact-us-desk.png"
                width="530"
                height="556"
                alt="watch"
              />
              </picture>

anakovalchuk reviewed (changes requested):
Перевір ще раз розмітку та стилі css. 
Для другого діва задані маржини і бекграунд, переконайся чи вони дійсно потрібні.

pavlovasofia65 committed:
6

pavlovasofia65 committed:
7

anakovalchuk reviewed (changes requested):
<img width="793" alt="Screenshot 2024-09-09 at 15 11 12" src="https://github.com/user-attachments/assets/db505fd4-d70b-4d5d-bd29-2d50a6979670">

Із зображенням ще попрацюй

pavlovasofia65 committed:
8

pavlovasofia65 committed:
9

anakovalchuk reviewed (approved):
Well done!!!
